diagnostics/TC_MSE/doc/TC_MSE.rst

```diff
@@ -0,0 +1,132 @@
+
+When and how each of the scripts are utilized in the driver script (TC_MSE_Driver.py) is as follows:
+
+1. TC_snapshot_MSE_calc.py is called first to extract the data and compute the MSE varaince budget along the tracks of all the TCs. The resulting data is saved into a file for each year. 
+
+2. Binning_and_compositing.py is called which takes all of the files that were created in step 1, concatenates them, and then bins as well as composites each
+of the snapshots and its variables by vmax. The budget variables are also box-averaged and normalized in this step as well.
+
+3. Plotting.py is called which imports all of the plotting functions that are in the Plotting_Functions.py script and generates and saves the plots that are 
+desired. The user may comment out any of the plotting functions that are called in the Plotting.py script they do not want. 
+
+Required programming language and libraries
+-------------------------------------------
+
+Written using conda version 4.10.1 and python version 3.10
+
+matplotlib, numpy, pandas, xarray, scipy, sys
+
+Required model output variables
+-------------------------------
+
+The following 3-D (time-lat-lon) and 4-D (time-plev-lat-lon) variables are required:
+
+3-D Variables:
+
+Surface Downwelling Longwave Flux in Air, Units: W m-2, Frequency: 6-hourly 
+Surface Downwelling Shortwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upwelling Longwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upwelling Shortwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upward Latent Heat Flux, Units: W m-2, Frequency: 6-hourly
+Surface Upward Sensible Heat Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Outgoing Longwave Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Outgoing Shortwave Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Incoming Shortwave Flux, Units: W m-2, Frequency: 6-hourly
+
+4-D Variables:
+
+Air Temperature, Units: K, Frequency: 6-hourly instantaneous
+Geopotential Height, Units: m, Frequency: 6-hourly instantaneous
+Specific Humidity, Units: g g-1, Frequency: 6-hourly instantaneous
+
+In order to have sufficient samples, we recommend enough years of data to have at least 100 TCs, which is typically 2-5 years of simulation time depending on the resolution of the model.
+
+In the current implementation of the POD, pre-calcluated TC track data is also required as an input to the POD as obs data. If future versions incorporate TC tracking directly, additional model output variables will be required.
```

Comment:
I missed the part where the GFDL data (is it AM4 or CM4??) is described and cited, associated with the Box average plots. If it's not cited, can you please describe and cite it appropriately. Thanks,

Reply:
Hello Aparna, I have committed changes to the .rst file to note the DOI for the model used in that example image. However, it is not part of our POD exclusively and it was just the sample model data that was used. That model will change depending on which model the user runs through our POD. Thank you!